### ISSUE TITLE
versioning system

### ISSUE THREAD

**Issue by Nilsonfsilva** [ansible/ansible-compat#376]
Hello"

I'm maintaining ansible-compat on Debian.

Let me ask:
The last update I made to Debian was v4.1.12

In the last release it went from v4.1.12 to v24.5.1. That's right? Or did ansible compat change the versioning system?

Would you be able to clarify?

Dear,
Nilson Silva

**Comment by ssbarnea** [comment]
Yes, we adopted CalVer versioning for it. The upgrade is safe, no breaking changes included, even if the versioning scheme did change.
